MonoGame.Framework.Content.Pipeline/TargetPlatform.cs

```diff
@@ -42,6 +43,7 @@ public enum TargetPlatform
         /// Linux-based PCs
         /// (MonoGame)
         /// </summary>
+        [Obsolete("This platform is obsolete, use DesktopGL instead")]
         Linux,
```

Comment:
Can we do like...
```
Linux = DesktopGL,
```
It would then at least take the same path as the desktop GL code.